SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -259,7 +260,26 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (DisplayName = "Auto-generate launch configuration file"))
 	bool bGenerateDefaultLaunchConfig;
 
+	/** Returns the Runtime version to use for cloud deployments, either the pinned one, or the user-specified one depending of the settings. */
+	const FString& GetSpatialOSRuntimeVersionForCloud() const;
+
+	/** Returns the Runtime version to use for local deployments, either the pinned one, or the user-specified one depending of the settings. */
+	const FString& GetSpatialOSRuntimeVersionForLocal() const;
+
+	/** Whether to use the GDK-associated SpatialOS runtime version, or to use the one specified in the RuntimeVersion field. */
+	UPROPERTY(EditAnywhere, config, Category = "Runtime", meta = (DisplayName = "Use GDK pinned runtime version"))
+	bool bUseGDKPinnedRuntimeVersion;
```

Comment:
nit: these all feel like `advanced` options and I'd recommend marking them so

Reply:
What do you mean by marking as "advanced" settings ? Putting them somewhere else ? Advanced mention in comments ?
That is also something that user will have to eventually deal with the moment they use the CLI from outside Unreal, so we might want to make it standard and visible.

Reply:
Configs can be tagged as `advanced` so they aren't shown by default.